docs/blog/2020-02-17-changing-the-oss-status-quo/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Changing the OSS Status Quo"
+date: 2020-02-17
+author: Aisha Blake
+excerpt: "How can we increase the diversity of open source contributors? As maintainers, we need to listen to a diverse group of community members and make thoughtful changes to create a truly inclusive project."
+tags:
+  - open-source
+  - community
+---
+
+In business and in our individual careers, so many of us rely heavily on open source software. We need it but open source isn't as open as it’s often made out to be. Too many projects are unfriendly, even openly hostile, to new contributors. This effect, combined with the intersection of many underrepresented identities in tech (BIPOC and women being some of the most visible), effectively shuts people out of the process. Consider who contributes to open source.
+
+The status quo is made and remade every time you as a maintainer add to your documentation. Every time you, as a maintainer, engage with the community via social media, issue queues, and your documentation, you help decide who contributes. You contribute to their safety and their productivity. Right now, barriers to contribution are severely limiting our capacity to create and sustain open source projects.
```

Comment:
Does it still make sense to frame this as an appeal to other maintainers? Does it still make sense if I switch to Gatsby devs, for example?

Reply:
I think it works either way. Blog posts have a more general reach than docs do, so opening it up to other maintainers makes sense to me.